api/src/main/java/org/openmrs/module/legacyui/api/impl/LegacyUIImpl.java

```diff
@@ -0,0 +1,291 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.legacyui.api.impl;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.Concept;
+import org.openmrs.Location;
+import org.openmrs.Obs;
+import org.openmrs.Order;
+import org.openmrs.Patient;
+import org.openmrs.PatientProgram;
+import org.openmrs.PatientState;
+import org.openmrs.ProgramWorkflow;
+import org.openmrs.ProgramWorkflowState;
+import org.openmrs.Provider;
+import org.openmrs.User;
+import org.openmrs.api.APIException;
+import org.openmrs.api.OrderService;
+import org.openmrs.api.ProviderService;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.impl.BaseOpenmrsService;
+import org.openmrs.module.legacyui.api.LegacyUIService;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * Default implementation of the patient service. This class should not be used on its own. The
```

Comment:
The description seems to be referring to a different service.

Reply:
[Fixed](https://github.com/openmrs/openmrs-module-legacyui/pull/128/commits/4753ae55d9f61c8dcc882fa4bb22bf60ec775b77)